baybe/searchspace/continuous.py

```diff
@@ -117,8 +119,11 @@ def param_names(self) -> List[str]:
         return [p.name for p in self.parameters]
 
     @property
-    def param_bounds_comp(self) -> torch.Tensor:
+    def param_bounds_comp(self) -> Tensor:
```

Comment:
remember me: did we agree to refactor functions that currently do but should not use tensors at all to a later PR?
Because this is def one of them, which means current changes are already obsolete soon